OnlTools/Jevp/StJevpBuilders/etofBuilder.h

```diff
@@ -14,6 +14,9 @@
 
 namespace gdpbv100 { class FullMessage; };
 
+#ifdef __CINT__
+typedef unsigned int uint64_t;
```

Comment:
But `uint64_t` is at least `unsigned long long`?!

Reply:
Yes, should be fixed

Reply:
I agree that the typedef is wrong.   What is the purpose of the change in the first place?  This code isn't ever run under CINT.

Reply:
The error is mentioned in the log message of this very commit https://github.com/star-bnl/star-sw/pull/101/commits/81e544306f9ef636b557e738e7cde6486a9de188. If this change is not needed as you think, just revert it and show that everything compiles without it.

Reply:
I've got no problem with the #ifdef __CINT__ code.   It will never execute in any situation I can imagine, but that's fine.    What I don't understand is what is being done and why.   I saw the message that things fail when you run with cint, but I don't know why this was tried or what was the intended effect.   Is the cint executation part of the autobuild?

I also don't understand what my role is in these updates.   The way it worked before, and the way, in my mind it should work is that if there is a problem with the autobuild for Jevp somebody should tell me and I'll fix it.   If not, hands off.  This is code that generates our QA plots online.   As mentioned before it needs to be in the autobuild because the code depends upon the offline chain, but the offline chain does not depend upon it.  

In terms of git, I thought I accepted these pull requests 2 weeks ago, so I don't have any idea what I am supposed to do now with regard to them.   I thought they were resolved.   Going through the history makes little sense to me.   Some comments are closed some are open.  Of these what do they mean.  "Star6 #59: open"?     Maybe this means something to somebody, but it means nothing to me.   Am I supposed to do something?  If so what?

Reply:
For motivation see #102 and #101. The CI build was failing for ROOT6. Not sure why #59 is relevant here

Reply:
I have no idea what is relevant here.   What is it that I am supposed to do?

Reply:
I don't know why this header is processed by rootcint. I showed you the error. This typedef seems to take care of the error but you'd better change `unsigned int` to `unsigned long long`

The error comes from the CI jobs setup with GitHub actions. The nightly builds on the farm do not build STAR sources against ROOT6 as far as I know. You can get in touch with the production team to confirm.

Reply:
Right, but this is not a patch I put in.   I can't even seem to see whose patch is.   Nor can I test against the root6 because I don't have instructions for that.    So I don't know what I'm supposed to be doing.

Reply:
> Right, but this is not a patch I put in. 

If it was a pull request, it can be found in the list of all PRs

> I can't even seem to see whose patch is.

Every commit has an author: plexoos committed 19 days ago commit 81e544306f9ef636b557e738e7cde6486a9de188
Not sure why you can't see this
This commit https://github.com/star-bnl/star-sw/pull/101/commits/81e544306f9ef636b557e738e7cde6486a9de188 belongs to PR #101 
 
> Nor can I test against the root6 because I don't have instructions for that.

This might work https://github.com/star-bnl/star-sw/discussions/68
If it does not create an issue. 
I personally test with docker images and that's why I am not sure about the bare environment on the farm. Things may not compile as, for example, reported in #139 There was no follow up, so I don't know if the proposed solution was adopted. This has to be confirmed with someone who manages the STAR environment, perhaps @jlauret

> So I don't know what I'm supposed to be doing.

For the trivial change such as replacing `unsigned int` with `unsigned long long` you can create a PR and the CI will be triggered automatically building the entire code base with both ROOT5 and ROOT6. There is even a few test jobs which will run on real events. The result will be displayed on the PR page when ready.